config/config_data_loader.ini

```diff
@@ -6,8 +6,6 @@ path_to_cloud=/samples/clouds
 cloud_extension=.pcd
 path_to_rgb=/samples/images
 rgb_extension=.png
-path_to_depth=/samples/depths
-depth_extension=.png
```

Comment:
I have not extensively tested this module yet, but if one has depth images shouldn't this stay? I presume you are modifying this for the unit tests, so I would recommend creating a separate data_loader config file for unit tests and not modifying the default one.